build/config/RepoUtilData.json

```diff
@@ -1,6 +1,16 @@
 ﻿{
   "fixedPackages": {
     "Microsoft.VSSDK.BuildTools": [ "14.3.25420" ],
+    "Microsoft.VisualStudio.CoreUtility": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Editor": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Text.Data": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Text.Logic": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Text.UI": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Text.UI.Wpf": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Utilities": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Shell.Framework": ["14.0.23205", "15.0.26014-alpha"],
+    "Microsoft.VisualStudio.Shell.14.0": ["14.0.23205"],
+    "Microsoft.VisualStudio.ComponentModelHost": ["14.0.23205", "15.0.26014-alpha"],
```

Comment:
Doesn't that meant the projects that are referencing the 15.0 packages will fail then?